R/mergeDat.R

```diff
@@ -48,19 +49,32 @@ toolMergeDat <- function(UCD_output, EU_data, PSI_costs, GDP_MER, altCosts, CHN_
 
   LF = rbind(LF_EU, GCAM_data$load_factor[!(iso %in% unique(LF_EU$iso) & vehicle_type %in% unique(LF_EU$vehicle_type))])
 
-  if (SSP_scen == "SDP_RC") {
+  ## set default targets for LF for LDVs
+  target_LF = 0
+  target_year = 2080
+
+  if(SSP_scen == "SDP_RC"){
     target_LF = 0.3
     target_year = 2060
-    LF[
-      subsector_L1 == "trn_pass_road_LDV_4W" &
+  }
+
+  if (!is.null(Dem_Scen)){
+    if (SSP_scen == "SSP2" & Dem_Scen == "SSP2EU_lowdem"){
+    target_LF = 0.4
```

Comment:
maybe put this into a config file (or put it on a todo list). Becoming a long list otherwise...

Reply:
Yes. But this is for the refactoring.. (there are already other adjustments in place and I have had not the time to clean up the implemented solution yet)